wp-admin/js/widgets/media-image-widget.js

```diff
@@ -31,6 +31,10 @@
 		 */
 		renderPreview: function renderPreview() {
 			var control = this, previewContainer, previewTemplate;
+			if ( ! control.model.get( 'attachment_id' ) && ! control.model.get( 'url' ) ) {
```

Comment:
Great question. I copied this statement from the `renderPreview` method in the video and audio widgets. Perhaps they should use `isSelected` as well…